tcp2canopsis/connector.py

```diff
@@ -53,11 +70,16 @@ def tryAuthentication(self, line):
                 self.address
             ))
 
-    def processLine(self, line):
+    processLine = None
```

Comment:
:camel: :warning: What is the use of a class member ? You should put it in the constructor.